helm/minio/values.yaml

```diff
@@ -104,7 +104,7 @@ existingSecret: ""
 
 ## Directory on the MinIO pof
 certsPath: "/etc/minio/certs/"
-configPathmc: "/etc/minio/mc/"
+configPathmc: "./minio/mc/"
```

Comment:
This is a breaking change, why is this being changed?

Reply:
> This is a breaking change, why is this being changed?
when mc client container use current system account do mc-job , ipermission denied will happen in dir `/etc/minio`.  so must be in this system account available  directory to do that shell command, so change it to` ./minio/mc/`

Reply:
such users can configure the mc config path properly, we shouldn't change the default values.